<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdListForm</class>
 <widget class="QWidget" name="ThresholdListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDateTimeEdit" name="dateTimeEditEnd">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>350</y>
     <width>194</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetLibrary">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="listWidgetThreshold">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>140</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetSpecial">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Algorithm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>設定時系列表示  /  設定値・しきい値の変更リスト表示</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEditStart">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>320</y>
     <width>194</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Special</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStartSearch">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>390</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>検索開始</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Library</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonView">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>表示</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>392</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>検索開始</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetAlgorithm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMasterName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>非選択で全マスターが対象</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>350</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>検索終了</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMasterID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectMaster">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>マスターデータ選択</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExcel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>470</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Excel出力</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
